src/ManagedNetworkFabric/ManagedNetworkFabric.Autorest/generate-info.json

```diff
@@ -1,3 +0,0 @@
-{
-  "generate_Id": "7e5c0295-7fb1-45f6-aecb-1a83e55f4a14"
```

Comment:
A new generate-info.json file should be generated after running build-module.ps1. Please include the new file in the PR.